src/usr/lib/rsetup/tui/system/system.sh

```diff
@@ -74,12 +74,13 @@ __system_bootloader_helper() {
         return
     fi
 
-    if "$bootloader_method" "$bootloader"
+    if __external_script_type_check "/usr/lib/u-boot/$bootloader/setup.sh" "$bootloader_method" "function" 2>/dev/null && \
```

Comment:
```suggestion
    if __external_script_type_check "/usr/lib/u-boot/$bootloader/setup.sh" "$bootloader_method" "function" && \
```
这个无所谓吧，就打印到stderr呗，反正一般也看不到